src/status_im/ui/screens/accounts/login/events.cljs

```diff
@@ -96,22 +96,26 @@
                         wrap-with-stop-node-fx)]
       (wrap-fn db' address password))))
 
+(defn is-new-account?
+  "checks if the account is in the accounts in the db"
+  [{:accounts/keys [accounts]} address]
+  (not ((into #{} (keys accounts)) address)))
+
 (register-handler-fx
   :login-handler
   (fn [{db :db} [_ result address]]
     (let [data    (json->clj result)
           error   (:error data)
           success (zero? (count error))
           db'     (assoc-in db [:accounts/login :processing] false)]
-      (log/debug "Logged in account: " result)
+      (log/debug "Logging result: " result)
```

Comment:
Can we preface this with login-handler so it is clear what result we are talking about?